### ISSUE TITLE
doctests are broken in pytest if hypothesis-3.5.1 is installed

### ISSUE THREAD

**Issue by kmike** [HypothesisWorks/hypothesis-python#367]
Hey,

After upgrade from hypothesis 3.4.2 to hypothesis 3.5.1 all doctests start to fail with the following error:

```
item = <DoctestItem 'eli5.sklearn.unhash._signed'>

    @pytest.mark.hookwrapper
    def pytest_runtest_call(item):
>       if not is_hypothesis_test(item.function):
E       AttributeError: 'DoctestItem' object has no attribute 'function'
```

I'm using pytest pytest-3.0.2.

**Comment by DRMacIver** [comment]
I can't seem to reproduce this. It's entirely possible I'm doing something wrong, as I don't use doctest myself, but when I try to run a doctest with Hypothesis installed it seems to work. Could you give me a small example to run that reproduces the problem?

**Comment by DRMacIver** [comment]
Sorry, no, never mind. I have a reproduction. I was just doing it wrong.

**Comment by DRMacIver** [comment]
If you upgrade to Hypothesis 3.5.2 this should be fixed.

**Comment by kmike** [comment]
Thanks for a quick fix @DRMacIver!
